README.md

```diff
@@ -15,9 +15,34 @@ Example
     $ rm -r 1.2
     $ ./add-version.sh -r 1.2 -f 1.2.1
 
+
+Stackbrew Manifest
+------------------
+
+`generate-stackbrew-library.sh` is used to generate the library file required for official Docker
```

Comment:
Excellent this documentation was so much needed!
if you agree I would like to deal maybe with the next release (e.g. 1.5.1 or 1.6) of the images since you have dealt with this most of the time.